tcf_website/views/search.py

```diff
@@ -208,22 +225,29 @@ def filter_courses(filters):
 
     return courses
 
+
 def apply_filters(results, filters):
```

Comment:
Is it more efficient to applies all three of these filters in a joined condition at once, rather than iteratively?

Speaking in terms of django under the hood.